playbooks/cluster_setup_network.yaml

```diff
@@ -58,8 +58,6 @@
   become: true
   hosts:
     - VMs
-  vars_files:
-    - ../vars/network_vars.yml
```

Comment:
if you remove this `vars_files`, you can remove this entire section "Apply network with systemd-networkd roles on VMs" (line 57-67).
But the admin IP address will not be set in the VM, we will have to define the .network file in the VM inventory.

Reply:
In order to play this task you need to access the VM. Thus admin IP is already configured (maybe with SEAPATH build_iso project) in VM disk and changing this configuration with a "br0" bridge is not something you need. However you might want to update the configuration or configure others NIC. To do so systemd_networkd role is the way to go, with variables like custom_network, systemd_networkd_link and custom_netdev.

Reply:
agree, but  `custom_network` and `custom_netdev` are configured and defined in this file `vars/network_vars.yml` so if you remove it, you can't use them anymore.

I think the best would be to create two new variables, may be `seapath_network` and `seapath_netdev`  that contains the entire network and netdev definition of the VM and that are passed to systemd_networkd role.
This would may be require a new vars file, but it would be way simpler than the previous one.